src/main/java/com/czertainly/api/model/core/settings/Oauth2SettingsDto.java

```diff
@@ -0,0 +1,19 @@
+package com.czertainly.api.model.core.settings;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+@Data
+public class Oauth2SettingsDto {
+
+    @Schema(
+            description = "Registration ID of Oauth2 Provider, the ID must be unique.",
+            example = "keycloak, github, facebook, etc.",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private String registrationId;
```

Comment:
Change to `providerName` or something more meaningful. This value will be shown to the user to select provider